bin/templates/scripts/cordova/Api.js

```diff
@@ -64,7 +64,17 @@ function Api (platform, platformRootDir, events) {
     var xcodeCordovaProj;
 
     try {
-        xcodeProjDir = fs.readdirSync(this.root).filter(function (e) { return e.match(/\.xcodeproj$/i); })[0];
+
+        var xcodeProjDir_array = fs.readdirSync(this.root).filter(function (e) { return e.match(/\.xcodeproj$/i); });
+        if (xcodeProjDir_array.length > 1) {
+            for (var x = 0; x < xcodeProjDir_array.length; x++) {
+                if (xcodeProjDir_array[x].substring(0, 2) === '._') {
+                    xcodeProjDir_array.splice(x, 1);
+                }
+            }
+        }
+        xcodeProjDir = xcodeProjDir_array[0];
+
```

Comment:
Thanks! I think this would be more efficient and clearer:
```javascript
var xcodeProjDir_array = fs
    .readdirSync(this.root)
    .filter(function (elem) {
        // match .xcodeproj, and not start with '._'
        return elem.match(/\.xcodeproj$/i) && (elem.substring(0, 2) !== '._'); 
    })

xcodeProjDir = xcodeProjDir_array[0];      
```